releases/v2.1.0-alpha.20180730.md

```diff
@@ -0,0 +1,263 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-alpha.20180730.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### General Changes
+
+- The [cost-based optimizer](../v2.1/sql-optimizer) is now on by default. [#26893][#26893]
+- The time series system used by CockroachDB to store internal monitoring data now utilizes pre-computed rollups to significantly increase the duration for which monitoring data is available while using less storage. Monitoring data will be available for up to a year by default; however, data older than seven days will be stored at a reduced resolution, and thus will only be able to give details about 30 minute intervals. [#27121][#27121] {% comment %}doc{% endcomment %}
+- Building CockroachDB from source now requires yarn at version 1.7.0 or above. [#27262][#27262] {% comment %}doc{% endcomment %}
+- Added support for signing server and client certificates by different CAs. [#27636][#27636] {% comment %}doc{% endcomment %}
+
+### Enterprise Edition Changes
+
+- Core dumps are now disabled and [encryption](../v2.1/encryption.html) is enabled. [#27426][#27426]
+- [`CHANGEFEED`](../v2.1/change-data-capture.html)s now use an asynchronous Kafka producer, increasing throughput. [#27421][#27421]
+
+### SQL Language Changes
+
+- CockroachDB now supports custom frame specification for [window functions](../v2.1/window-functions.html) using `ROWS` (fully-supported) and `RANGE` modes. For `RANGE`, `<value> PRECEDING` and `<value> FOLLOWING` are not supported. [#26666][#26666] {% comment %}doc{% endcomment %}
+- The `SNAPSHOT` [isolation level](../v2.1/transactions.html#isolation-levels) has been removed. Transactions that request to use it are now mapped to `SERIALIZABLE`. [#27040][#27040]. {% comment %}doc{% endcomment %}
+- When the cost-based optimizer is enabled, it will also affect prepared queries. [#27034][#27034]
+- Upon failing to gather data from other nodes, the [`SHOW CLUSTER QUERIES`](../v2.1/show-queries.html) and [`SHOW CLUSTER SESSIONS`](../v2.1/show-sessions.html) statements now report the details of the error. [#26821][#26821]
+- Improved the description for the `age()` [built-in function](../v2.1/functions-and-operators.html). [#27082][#27082] {% comment %}doc{% endcomment %}
+- The `pg_get_indexdef()` built-in function now supports 3 arguments. [#27161][#27161] {% comment %}doc{% endcomment %}
+- Added `COPY` support to `IMPORT .. . PGDUMP`. [#27062][#27062] {% comment %}doc{% endcomment %}
+- The new `max_row_size` option overrides default limits on line size for `IMPORT ... PGDUMP` and `PGCOPY`. [#27062][#27062] {% comment %}doc{% endcomment %}
+- The `SHOW TRACE FOR <stmt>` statement was incomplete and incorrect and has thus been removed. To use tracing, use `SET tracing = ...` and `SHOW SESSION TRACE` instead. [#26729][#26729] {% comment %}doc{% endcomment %}
```

Comment:
Perhaps also make a cross-reference to the corresponding CLI change (`auto_trace`).

Reply:
Done.